endpoint/endpoint.go

```diff
@@ -28,11 +28,11 @@ func Process[T any](ctx context.Context, name string, endpoint *Endpoint, handle
 					return &r, nil
 				} else {
 					var buff []byte
-					buff, err = json.Marshal(resp)
+					buff, err = jsoniter.Marshal(resp)
```

Comment:
We couldn't fully remove `encoding/json` library because of some missing features of the `jsoniter`. Since both libraries coexist in the repo, I wanted to explicitly provide the library name to prevent careless maintenance changes.